### DISCUSSION TITLE
As I changed my Raspberry Pi IP, some links have become invalid. What should I do?

### DISCUSSION THREAD

**Discussion by AemonCao** [teslamate-org/teslamate#1895] [Q&A] [answered]
I use [docker](https://docs.teslamate.org/docs/installation/docker) to install.

The original IP address of my Raspberry Pi is: **192.168.2.121**.

When I click on the address in the Start column or Destination column on this page, it will jump to the address <http://192.168.2.121:4000/geo-fences/1/edit>.

![image](https://user-images.githubusercontent.com/19542659/128705392-6e016eb0-6e4e-47a8-b1e0-1719d85b5701.png)

Now I changed the IP address of my Raspberry Pi to **192.168.1.144**. After clicking the same link, I still jumped to the original address, causing the access to fail.

How should I modify the link address to: <http://192.168.1.144:4000/geo-fences/1/edit>.

**Comment by DrMichael** [comment, marked as answer]
You might want to change it in the teslamate preferences...

**Reply by SteveDonie** [reply]
Seconded - at the bottom of the settings page are two boxes for the URLs. They get pre-populated at install time I think, so if you change after install time they will be incorrect.
![image](https://user-images.githubusercontent.com/605418/128731999-7abcb5a1-f681-4b5a-887a-26942461c609.png)

**Reply by AemonCao** [reply]
Thanks!

**Reply by AemonCao** [reply]
> Seconded - at the bottom of the settings page are two boxes for the URLs. They get pre-populated at install time I think, so if you change after install time they will be incorrect.
> ![image](https://user-images.githubusercontent.com/605418/128731999-7abcb5a1-f681-4b5a-887a-26942461c609.png)

Thanks!
